extensions/amp-experiment/amp-experiment.md

```diff
@@ -16,4 +16,90 @@ limitations under the License.
 
 # <a name="amp-experiment"></a> `amp-experiment`
 
-Under development. See [issue #1411](https://github.com/ampproject/amphtml/issues/1411).
+| | |
```

Comment:
That's an [unfortunate](http://stackoverflow.com/questions/17536216/create-table-without-header-in-markdown)... 

If this really annoys you, I can switch to HTML `<table>` ..